### ISSUE TITLE
sidekiq/cli is broken on Ruby 2.5 & Sidekiq 6.5.2+

### ISSUE THREAD

**Issue by Fryguy** [mperham/sidekiq#5460]
Ruby version: 2.5.9
Rails version: N/A
Sidekiq / Pro / Enterprise version(s): 6.5.3

```
$ irb
irb(main):001:0> require "sidekiq/cli"
Traceback (most recent call last):
       16: from (irb):1
       15: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:39:in `require'
       14: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
       13: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:135:in `require'
       12: from /Users/jfrey/.gem/ruby/2.5.9/gems/sidekiq-6.5.3/lib/sidekiq/cli.rb:13:in `<top (required)>'
       11: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
       10: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        9: from /Users/jfrey/.gem/ruby/2.5.9/gems/sidekiq-6.5.3/lib/sidekiq/launcher.rb:3:in `<top (required)>'
        8: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        7: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        6: from /Users/jfrey/.gem/ruby/2.5.9/gems/sidekiq-6.5.3/lib/sidekiq/manager.rb:3:in `<top (required)>'
        5: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        4: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        3: from /Users/jfrey/.gem/ruby/2.5.9/gems/sidekiq-6.5.3/lib/sidekiq/processor.rb:5:in `<top (required)>'
        2: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
        1: from /Users/jfrey/.rubies/ruby-2.5.9/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
SyntaxError (/Users/jfrey/.gem/ruby/2.5.9/gems/sidekiq-6.5.3/lib/sidekiq/job_retry.rb:214: void value expression)
      end
         ^
irb(main):002:0> RUBY_VERSION
=> "2.5.9"
irb(main):003:0> Sidekiq::VERSION
=> "6.5.3"
```

It works fine with 6.5.1, so this was introduced in 6.5.2:

```
$ irb
irb(main):001:0> require "sidekiq/cli"
=> true
irb(main):002:0> RUBY_VERSION
=> "2.5.9"
irb(main):003:0> Sidekiq::VERSION
=> "6.5.1"
```

**Comment by mperham** [comment]
I can't even compile Ruby 2.5.9 on my MBP M2 anymore. Can you upgrade Ruby?

**Comment by mperham** [comment]
Ok, github actions still supports Ruby 2.5. I've reproduced the issue and rejiggered the code to work. Can you verify `main` works for you?

**Comment by Fryguy** [comment]
main also fails in the same way

**Comment by mperham** [comment]
I don't understand how that's possible unless you have an old clone of the repo. The test suite was failing and now passes with my fix.

**Comment by Fryguy** [comment]
oh I had a clone from 30 minutes ago - didn't realize you pushed a fix - let me try again

**Comment by Fryguy** [comment]
main is fixed after acdaa85b5577dfb1a7565d59a8715ea5d0d029af

**Comment by fotinakis** [comment]
@mperham I see 6.5.4 is in the changelog with this fix but looks like it's not on rubygems yet. Can you please publish the gem as well? Thank you!

**Comment by mperham** [comment]
Thanks for the reminder. I like to let `main` sit for a few days before rushing another release. In this case it also picked up a fix for Ruby 2.5.
